### ISSUE TITLE
Error in Paring Device with QR

### ISSUE THREAD

**Issue by bhavesh53** [tulir/whatsmeow#584]
i am getting this error after scan QR . 

09:43:18.677 [Client INFO] Successfully paired
09:43:18.880 [Client ERROR] Unknown stream error: <stream:error code="516"/>
09:43:18.976 [Client/Socket ERROR] Error reading from websocket: websocket: close 1006 (abnormal closure): unexpected EOF

**Comment by TanvirSingh007** [comment]
I am also not able to pair new devices due to this error.

**Comment by JustSoursop** [comment]
I've tested the latest [mdtest](https://github.com/tulir/whatsmeow/blob/01b0547014dc47a63ddce5c1287f64c7d1dc26a4/mdtest/main.go) and had successed login via qr code.

Actually I've same case yesterday but unable to figure it out. Today I'd tested the same code and it works fine.

**Comment by RakeshBhaiKush** [comment]
> I am also not able to pair new devices due to this error.

Hello,

Could we possibly connect via Google Meet or another platform? I'm encountering some problems with Git installation.

**Comment by TanvirSingh007** [comment]
> > I am also not able to pair new devices due to this error.
> 
> Hello,
> 
> Could we possibly connect via Google Meet or another platform? I'm encountering some problems with Git installation.

It is out of scope of this thread. However, you can read the documentation to install git. It is pretty straightforward.

**Comment by RakeshBhaiKush** [comment]
> It is out of scope of this thread. However, you can read the documentation to install git. It is pretty straightforward.

Sorry for this its done , you are still facing scanning issue ?

**Comment by TanvirSingh007** [comment]
Yes, I am still facing the issue. I am facing the same issue with whatsapp-web.js, which is a javascript library to implement the same functionality.

**Comment by RakeshBhaiKush** [comment]
> Yes, I am still facing the issue. I am facing the same issue with whatsapp-web.js, which is a javascript library to implement the same functionality.

Do you think this issue happens after the latest whatsapp update !! or it is anyother reason !!

**Comment by TanvirSingh007** [comment]
I think its due to some update to whatsappweb from Meta as the functionality seems to be breaking in different regions at different times, across different libraries.

**Comment by RakeshBhaiKush** [comment]
> I think its due to some update to whatsappweb from Meta as the functionality seems to be breaking in different regions at different times, across different libraries.

Oh by any change how much time it gonna take !!

**Comment by TanvirSingh007** [comment]
IDK

**Comment by rajuaryan21** [comment]
However on local machine this works but not on cloud server.

**Comment by rajuaryan21** [comment]
check this thread discussion here: https://github.com/WhiskeySockets/Baileys/issues/779

**Comment by TanvirSingh007** [comment]
> check this thread discussion here: https://github.com/WhiskeySockets/Baileys/issues/779

I believe I am getting the same issue.

**Comment by RakeshBhaiKush** [comment]
Any update guys

**Comment by AyushmanChatterjee** [comment]
Getting the same issue. I am also trying to pair an Indian number.

**Comment by TanvirSingh007** [comment]
Has anyone tried to use proxies to solve this problem?

**Comment by AyushmanChatterjee** [comment]
Do you have any working proxy server details? I can test it but it seems I might need to setup a proxy server myself for testing.

**Comment by nnarayanmuchhal** [comment]
We've tested it with residential proxies with that it works...
But it has become very costly affair.. 
If we keep lesser number of IP's and pay as per IP's then chances are banning goes up 
if we keep rotating IP's then the cost of data transfer is too high.. so it is too difficult to maintain the cost and sell services at lower price..

**Comment by TanvirSingh007** [comment]
> We've tested it with residential proxies with that it works... But it has become very costly affair.. If we keep lesser number of IP's and pay as per IP's then chances are banning goes up if we keep rotating IP's then the cost of data transfer is too high.. so it is too difficult to maintain the cost and sell services at lower price..

I was looking into using residential proxies for my use and I was wondering about the costs. I believe we don't have any better solution for the problem at the moment.

**Comment by RakeshBhaiKush** [comment]
I have tried with some VPN some numbers are getting connected some not

**Comment by ahmedRSA** [comment]
how this issue related to whatsmeow ?
transfer this issue to discussion

**Comment by alsak0de** [comment]
https://github.com/tulir/whatsmeow/issues/590#issuecomment-2122813237
